Shevchuk_Maksym/README.md

```diff
@@ -0,0 +1,93 @@
+Discord Registration and Usage Guide
+Introduction
+Discord is a popular communication platform that enables users to chat, voice call, and video call with individuals and groups. This document provides a step-by-step guide on how to register and use Discord effectively.
+
+Table of Contents
+Creating a Discord Account
+Setting up Discord
+Joining a Server
+Navigating the Discord Interface
+Sending Messages
+Voice and Video Calls
+Managing Notifications
+Customizing Discord
+Privacy and Security
+Troubleshooting
+1. Creating a Discord Account
```

Comment:
Use sentence style capitalization for the headings. That is, capitalize the first word and lowercase the rest.
https://learn.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings